http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -819,19 +823,60 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       }
     } yield ()
 
-  private[leonardo] def buildGalaxyPostgresDisk(zone: ZoneName, namespaceName: NamespaceName): Disk =
+  private[leonardo] def buildGalaxyPostgresDisk(zone: ZoneName, dataDiskName: DiskName): Disk =
     Disk
       .newBuilder()
-      .setName(getGalaxyPostgresDiskName(namespaceName).value)
+      .setName(getGalaxyPostgresDiskName(dataDiskName).value)
       .setZone(zone.value)
       .setSizeGb(config.galaxyDiskConfig.postgresDiskSizeGB.gb.toString)
       .setPhysicalBlockSizeBytes(config.galaxyDiskConfig.postgresDiskBlockSize.bytes.toString)
       .putAllLabels(Map("leonardo" -> "true").asJava)
       .build()
 
-  private[leonardo] def getGalaxyPostgresDiskName(namespaceName: NamespaceName): DiskName =
+  private[leonardo] def getGalaxyPostgresDiskName(dataDiskName: DiskName): DiskName =
+    DiskName(s"${dataDiskName.value}-${config.galaxyDiskConfig.postgresDiskNameSuffix}")
+
+  private[leonardo] def getOldStyleGalaxyPostgresDiskName(namespaceName: NamespaceName): DiskName =
```

Comment:
Maybe should add a TODO to delete the `oldStyle` method once there are no legacy Galaxy disks in the system>

Reply:
yea...I think that'd be a good ticket to do

Reply:
Ticket sounds good 👍

Reply:
I'll make a ticket👍